src/main/groovy/org/pih/warehouse/inventory/CycleCountCustomItemCommand.groovy

```diff
@@ -0,0 +1,41 @@
+package org.pih.warehouse.inventory
+
+import grails.databinding.BindUsing
+import grails.validation.Validateable
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.product.Product
+import org.springframework.web.context.request.RequestContextHolder
+
+class CycleCountCustomItemCommand implements Validateable {
```

Comment:
Same comment from Evan's PR (and below) about naming convention. Let's either go with a consolidated CycleCountItemCommand (that can be used for update, create, etc). Or an action-based command. In this case I don't think "custom" is an important distinction. It's just another cycle count item.